coordinator/persistence-cassandra-4.1/src/main/java/io/stargate/db/cassandra/impl/AuthenticatorWrapper.java

```diff
@@ -0,0 +1,63 @@
+package io.stargate.db.cassandra.impl;
+
+import io.stargate.db.AuthenticatedUser;
+import io.stargate.db.Authenticator;
+import java.net.InetAddress;
+import javax.security.cert.X509Certificate;
+import org.apache.cassandra.auth.IAuthenticator;
+import org.apache.cassandra.stargate.exceptions.AuthenticationException;
+
+public class AuthenticatorWrapper implements Authenticator {
```

Comment:
I know these have not been documented but was wondering if Javadoc on wrapping makes sense -- looks like here we really do it to convert Cassandra `AuthenticationException`.

But since this is probably the same reason for most types might not be worth adding it on every single class.